Add passive node (#4008)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/features/clustering-features.md

version 0.7.1.
See [Backup and restore](/enterprise_influxdb/v1.9/administration/backup-and-restore/) for
more information.

## Passive node setup (experimental)

Passive nodes are data nodes that do not own shards.  They accept write calls but do not write any data to their own disks.  They perform shard lookup and RPC calls (on non-passive nodes) and distribute writes to the data nodes that own shards.  In essence, they act as a kind of load balancer. You can use this feature when you have a replication factor (RF) of 2 or more and your CPU usage is consistently above 80 percent.  Using the passive feature allows you to scale a cluster when you can no longer vertically scale. This active-passive node setup feature is especially useful if you experience a large amount of hinted handoff growth. The passive node writes the hinted handoff queue to its own disk and then communicates periodically with the appropriate node until it can send the queue contents there.

Best practices when using an active-passive node setup: 
  - Use when you have a large cluster setup, generally 8 or more nodes.

>   - You should have an equal number of active and passive nodes.  

Reviewer: "You should have between 1:1 and 2:1 ratio of active to passive nodes"

Further testing since we spoke......

Author: made suggested change - thank you!

version 0.7.1.
See [Backup and restore](/enterprise_influxdb/v1.9/administration/backup-and-restore/) for
more information.

## Passive node setup (experimental)

> Passive nodes are data nodes that do not own shards.  They accept write calls but do not write any data to their own disks.  They perform shard lookup and RPC calls (on non-passive nodes) and distribute writes to the data nodes that own shards.  In essence, they act as a kind of load balancer and write reprocessor which will improve performance in your cluster.

Reviewer 2: ```suggestion
Passive nodes act as load balancers--they accept write calls, perform shard lookup and RPC calls (on active data nodes), and distribute writes to active data nodes. They do not own shards or accept writes.
```

>  You can use this feature when you have a replication factor (RF) of 2 or more and your CPU usage is consistently above 80 percent.  Using the passive feature allows you to scale a cluster when you can no longer vertically scale. This active-passive node setup feature is especially useful if you experience a large amount of hinted handoff growth. The passive node writes the hinted handoff queue to its own disk and then communicates periodically with the appropriate node until it can send the queue contents there.  

Reviewer 2: ```suggestion
Use this feature when you have a replication factor (RF) of 2 or more and your CPU usage is consistently above 80 percent. Using the passive feature lets you scale a cluster when you can no longer vertically scale. Especially useful if you experience a large amount of hinted handoff growth. The passive node writes the hinted handoff queue to its own disk, and then communicates periodically with the appropriate node until it can send the queue contents there.  
```

Best practices when using an active-passive node setup: 
  - Use when you have a large cluster setup, generally 8 or more nodes.

>   - You should have between a 1:1 and 2:1 ratio of active and passive nodes.

Reviewer 2: ```suggestion
  - Keep the ratio of active to passive nodes between 1:1 and 2:1.
```

>   - The passive nodes should receive all writes.  

Reviewer 2: ```suggestion
  - Passive nodes should receive all writes.  
```

> For instructions on how to add a paasive node to a cluster see [Add a passive node to a cluster](/enterprise_influxdb/v1.9/tools/influxd-ctl/#add-a-passive-node-to-the-cluster).

Reviewer 2: ```suggestion
For more inforrmation, see how to [add a passive node to a cluster](/enterprise_influxdb/v1.9/tools/influxd-ctl/#add-a-passive-node-to-the-cluster).
```

{{% note %}}

> **NOTE:**  This feature is experimental and available only in Influx Enterprise.

Reviewer 2: ```suggestion
**Note:**  This feature is experimental and available only in InfluxDB Enterprise.
```
